indico/htdocs/sass/modules/_papers.scss

```diff
@@ -39,3 +39,23 @@
         }
     }
 }
+
+#assign-role-table {
```

Comment:
I think this is the best approach we can follow. Reusing components will save us some headaches and make our codebase cleaner.